proxy.go

```diff
@@ -43,7 +45,13 @@ var crashOnError = false
 var proxyLog = logrus.New()
 
 func serve(servConn io.ReadWriteCloser, proto, addr string, results chan error) (net.Listener, error) {
-	session, err := yamux.Client(servConn, nil)
+	sessionConfig := yamux.DefaultConfig()
+	sessionConfig.EnableKeepAlive = true
+	// 10 seconds is too early to suspect that a problem with the underlying connection has occurred.
+	// Since we don't know how much time a container can be paused let's set this timeout to 100 years,
+	// probably we won't be there when that occurs ;(
+	sessionConfig.ConnectionWriteTimeout = year * 100
```

Comment:
Ha! That made my day! :smile:

Reply:
LOL !

@devimc is there any way to disable this timeout ? Something like providing 0 or -1 ?

Reply:
hahahaha - dahh - good point, let me try  xD